docs/architecture/fabric.md

```diff
@@ -1,92 +1,94 @@
+A collapsed core topology is just a pair of switches connected in a MCLAG configuration with no other network elements.
 All workloads attach to these two switches.
 
 ![image](./fabric-collapsedcore.png)
 
-The leaf's in this setup are configured to be in a mclag pair and servers can either be connected to both switches as
-a mclag port channel or as orphan ports connected to only one switch. both the leaves peer to external networks using
+The leaves in this setup are configured to be in a MCLAG pair and servers can either be connected to both switches as
+a MCLAG port channel or as orphan ports connected to only one switch. Both the leaves peer to external networks using
 BGP and act as gateway for workloads attached to them. The configuration of the underlay in the collapsed core is very
 simple and is ideal for very small deployments.
 
-### Spine - Leaf
+### Spine-Leaf
 
-A spine-leaf topology is a standard clos network with workloads attaching to leaf switches and spines providing
+A spine-leaf topology is a standard Clos network with workloads attaching to leaf switches and the spines providing
 connectivity between different leaves.
 
 ![image](./fabric-spineleaf.png)
 
-This kind of topology is useful for bigger deployments and provides all the advantages of a typical clos network.
+This kind of topology is useful for bigger deployments and provides all the advantages of a typical Clos network.
 The underlay network is established using eBGP where each leaf has a separate ASN and peers will all spines in the
-network. We used [RFC7938](https://datatracker.ietf.org/doc/html/rfc7938) as the reference for establishing the
+network. [RFC7938](https://datatracker.ietf.org/doc/html/rfc7938) was used as the reference for establishing the
 underlay network.
 
 ## Overlay Network
 
 The overlay network runs on top the underlay network to create a virtual network. The overlay network isolates control
 and data plane traffic between different virtual networks and the underlay network. Visualization is achieved in the
-hedgehog fabric by encapsulating workload traffic over vxlan tunnels that are source and terminated on the leaf switches
-in the network. The fabric using BGP-EVPN/Vxlan to enable creation and management of virtual networks on top of the
-virtual. The fabric supports multiple virtual networks over the same underlay network to support multi-tenancy. Each
-virtual network in the hedgehog fabric is identified by a VPC. In the following sections we will dive a bit deeper into
-a high level overview of how are vpc's implemented in the hedgehog fabric and it's associated objects.
+Hedgehog Fabric by encapsulating workload traffic over VXLAN tunnels that are source and terminated on the leaf switches
+in the network. The fabric uses BGP-EVPN/VXLAN to enable the creation and management of virtual networks on top of the
+physical one. The fabric supports multiple virtual networks over the same underlay network to support multi-tenancy.
```

Comment:
> The fabric using BGP-EVPN/Vxlan to enable creation and management of virtual networks on top of the virtual.

I believe it was meant to be “on top of the physical” rather than “virtual” at the end of this sentence.